spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1369,16 +1363,23 @@ def _do_load(self):
         # algorithms needed for loading the binaries to the SpiNNaker machine
         optional_algorithms.append("FrontEndCommonLoadExecutableImages")
 
+        # Get the executable targets
+        optional_algorithms.append("FrontEndCommonGraphBinaryGatherer")
```

Comment:
I just mean, swap line 1364 with this one (and the comments), so that the order is correct (gather then load).